package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@licenserocks/kit",
-  "version": "3.4.1",
+  "version": "3.4.104",
   "description": "",
```

Comment:
as we add manually the version, should we progress the number like 3.4.2 😅 ?

Reply:
ye i forgot git push --force :P